tests/test_level/test_level_05.py

```diff
@@ -454,16 +454,6 @@ def test_if_print_has_no_turtle(self):
     if computerkeuze is jouwkeuze print 'gelijkspel!'""")
     result = hedy.transpile_inner(code, self.level)
     self.assertEqual(False, result.has_turtle)
-  def test_no_space_after_keyword_gives_missing(self):
```

Comment:
I think we have an issue open for this (this too is a note to self)

Reply:
Ah here is the (closed) issue: https://github.com/Felienne/hedy/issues/857

But I see that for some reason we are not allowing a space after _IS, is that intentional @jpelay (I can imagine it leads to all sorts of issues!)

Reply:
I didn't think about it, I followed closely the grammar that was already in place. I can try changing it and see if it doesn't raise weird issues!